security/libu2f-host/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                libu2f-host
+version             1.1.3
+categories          security
+platforms           darwin
+maintainers         nomaintainer
```

Comment:
I think the general policy is against accepting nomaintainer submissions unless it's a dependency of a maintained port